src/main/java/gov/nasa/pds/tools/validate/content/table/FieldValueValidator.java

```diff
@@ -422,7 +422,7 @@ private void checkMinMax(String value, Double minimum, Double maximum,
         }
 
         LOG.debug("checkMinMax:originalValue,value,minimum,maximum {},{},{},{}",originalValue,value,minimum,maximum);
-        System.out.println("checkMinMax:originalValue,value " + originalValue + " " + value);
+        //System.out.println("checkMinMax:originalValue,value " + originalValue + " " + value);
```

Comment:
can we just remove this code instead of commenting it out? i think we have enough lines of code in this software to remove this :-)